src/nvim/message.c

```diff
@@ -2234,15 +2234,11 @@ void mch_errmsg(char *str)
 {
   int len;
 
-#if (defined(UNIX) || defined(FEAT_GUI)) && !defined(ALWAYS_USE_GUI)
+#if defined(UNIX)
   /* On Unix use stderr if it's a tty.
    * When not going to start the GUI also use stderr.
    * On Mac, when started from Finder, stderr is the console. */
-  if (
-# ifdef UNIX
-    isatty(2)
-# endif
-    ) {
+  if (isatty(2)) {
```

Comment:
> without any other check ?

What do you mean? I just mean replace the two instances of `isatty` in this PR with `os_isatty`, since they have the same semantics AFAICT:
```
/// Test whether a file descriptor refers to a terminal.
///
/// @param fd File descriptor.
/// @return `true` if file descriptor refers to a terminal.
bool os_isatty(int fd)
{
    return uv_guess_handle(fd) == UV_TTY;
}

```